.github/workflows/check-cherry-pick.yml

```diff
@@ -0,0 +1,19 @@
+name: Check Cherry Pick
+
+on:
+  pull_request:
+    types: [synchronize, opened, labeled]
+
+jobs:
+  rebase-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v1
+      - name: Check Cherry Pick
+        if: >
+          contains(github.pull_request.labels.*.name, 'PR target: master & patch') ||
+          contains(github.pull_request.labels.*.name, 'PR target: patch')
```

Comment:
I don't know about other repos, but there is no such label on angular/angular :grin: (For `PR target:` we have `master & patch`, `master-only`, `patch-only`, `LTS-only` and `TBD`.) 

Also, why limit this to PRs with specific `PR target` labels? Is it to save resources?

Reply:
If it wasn't designed to be cherry-picked onto a different branch, there is a very high chance it will fail.  At least this is the case for the CLI.  With that in mind, testing something that you know will fail would both waste resources and give a false impression as to the viability of the PR (someone would then need to explain that failing here doesn't actually matter for this case).

Separately, there's also the consideration that there can be multiple LTS versions active at any point in time.  Currently both 6.x and 7.x are LTS (https://angular.io/guide/releases#support-policy-and-schedule).

Reply:
Ooops, I somehow thought this was about rebasing on the target branch 😇
For checking "cherry-pick-ability" having `PR target: master & patch` and `PR target: patch-only` makes sense. (Notice that it is `patch-only`, not `patch`, though.)

Reply:
So this yaml file is actually just the definition for the workflow in this repo, not in the repos where we actually use the action such as angular/angular, angular/components, angular/angular-cli, etc.

Its only really here as an illustration.